src/app/searchResults/searchResults.spec.js

```diff
@@ -57,12 +56,13 @@ describe('searchResults', function () {
     })
 
     it('pulls ministry list', () => {
-      jest.spyOn($ctrl.designationsService, 'productSearch').mockReturnValue(Observable.of([]))
+      jest.spyOn($ctrl.designationsService, 'ministriesList').mockReturnValue(Observable.of([]))
```

Comment:
The test might be more interesting if there was at least one result instead of an empty array.

Reply:
Added one result to test